Projects/Dotmim.Sync.MySql/Builders/MySqlDbMetadata.cs

```diff
@@ -439,6 +447,8 @@ public override bool IsTextType(string typeName)
             string lowerType = typeName.ToLowerInvariant();
             switch (lowerType)
             {
+                case "varbinary":
+                case "binary":
```

Comment:
varbinary / binary are considered as text in mysql ?

Reply:
The documentation says

> The BINARY and VARBINARY types are similar to CHAR and VARCHAR, except that they store binary strings rather than nonbinary strings. That is, they store byte strings rather than character strings. This means they have the binary character set and collation, and comparison and sorting are based on the numeric values of the bytes in the values.

They store binary strings, and comparison and sorting are based on the numeric values of the bytes in the values

[Docs](https://dev.mysql.com/doc/refman/8.0/en/binary-varbinary.html)

Reply:
Ok, Sounds good to me !